FirebaseAppDistribution/Tests/Unit/FIRAppDistributionTests.m

```diff
@@ -307,6 +313,8 @@ - (void)testFetchNewLatestReleaseSuccess {
 - (void)testFetchNewLatestReleaseNoNewRelease {
   [self mockFetchReleasesCompletion:_mockReleases error:nil];
   OCMStub([_mockMachO codeHash]).andReturn(_mockCodeHash);
+  OCMStub([[self appDistribution] getAppVersion]).andReturn(_mockDisplayVersion);
```

Comment:
You can also stub the main bundle directly with a class mock, but this works too 😄